modules/core/src/v2/coins/eosutil/eosabiprovider.ts

```diff
@@ -0,0 +1,32 @@
+const eosioNullAbi = (new Api({ textDecoder: new TextDecoder(), textEncoder: new TextEncoder() })).jsonToRawAbi(JSON.parse('{"version":"eosio::abi/1.1","types":[],"structs":[],"actions":[{"name":"nonce","type":"string","ricardian_contract":"---\\nspec_version: \\"0.2.0\\"\\ntitle: Nonce\\nsummary: Nonce to prevent duplicate transctions"}],"tables":[],"ricardian_clauses":[],"error_messages":[],"abi_extensions":[],"variants":[]}'));
+
+interface EosRawAbi {
+  accountName: string;
+  abi?: Uint8Array
+}
+
+export class OfflineAbiProvider {
+  getRawAbi(account: string): EosRawAbi {
+    const ret = {
+      accountName: account,
+      abi: undefined,
+    };
+    if (account === 'eosio') {
+      ret.abi = eosioAbi;
+    } else if (account === 'eosio.token') {
+      ret.abi = eosioTokenAbi;
+    } else if (account === 'eosio.null') {
+      ret.abi = eosioNullAbi;
+    } else {
+      throw new Error(`${account} is not supported by OfflineAbiProvider`);
+    }
```

Comment:
let's define a dictionary for supported accounts.
something like:
```
SUPPORTED_ABIS = {
    'eosio': ...,
    'eosio.token': ...,
    ...
}
```

error is thrown when `account` is not present in the dict.
this will help us trim down the if-else checks as we support new ABIs (which we will need for EOS tokens)